flags.css

```diff
@@ -86,7 +86,26 @@
     background-color: red
 }
 /* Sweden */
-
+#sweden {
+    position: relative;
+    background-color: rgb(0, 132, 255)
+}
+#sweden>.bar {
+    position: absolute;
+    background-color: rgb(255, 217, 0)
+}
+#sweden>.vertical {
+    width: 12.5%;
```

Comment:
if you use height and width based on %, then hour bars will have a chance of being the wrong size as they scale.  That's why I use vmin in mine.